learn/configuration/instance_options.md

```diff
@@ -4,10 +4,101 @@ sidebarDepth: 2
 
 # Configure Meilisearch at launch
 
-You can configure Meilisearch at launch with **environment variables** and **command-line options**.
+You can configure Meilisearch at launch with **command-line options**, **environment variables**, or a **configuration file**.
 
 These startup options affect your entire Meilisearch instance, not just a single index. For settings that affect search within a single index, see [index settings](/learn/configuration/settings.md).
 
+## Configuration file
+
+Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
+
+**Environment variables can overwrite the configuration file, and command-line options can overwrite environment variables.**
```

Comment:
I would add some context, like "using environment variables and config file at the same time would lead to..."